Userland/Libraries/LibGLSL/Lexer.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2020, Stephan Unverwerth <s.unverwerth@serenityos.org>
```

Comment:
Did stephan write this file?

Reply:
He wrote the LibJS Lexer.h which this is based on.